mobify/progressive-web-scaffold
# Refactor Footer

Refactor the home page's skeletons (and a few other things) so the page loads more seamlessly.

## Changes
- Refactor Footer container
- Tweak the footer skeletons to more closely match their final content

| Before | After |
| ------ | ------ |
| ![skeletons-before](https://cloud.githubusercontent.com/assets/734535/20612173/0092264a-b26c-11e6-837e-aaaab09471b0.gif) | ![skeletons-after](https://cloud.githubusercontent.com/assets/734535/20612175/06b5750e-b26c-11e6-9ee9-dfcf0b210be8.gif) |

> Notice how the subscription form now looks like a form the entire time!
>
> Note that the animated gifs are compressed, so the skeleton animations may not appear as smooth as they would in reality.

## How to test-drive this PR
- Run `npm run dev`
- [Preview Merlin's Potions](https://preview.mobify.com/?url=http%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Observe the footer while the page is still loading
- Confirm that the skeletons to real-content transition is smooth
